### PR TITLE
Submit assignments

### PR DESCRIPTION
**Pull request by pythonwa**
Submit second assignment and FizzBuzz

### CODE REVIEW

File: students/tonylee/FizzBuzz.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+def printFizzBuzz():
```
Comment (charlesrobison) [inline students/tonylee/FizzBuzz.py]: Per PEP8 functions should be lower case and words separated by underscores.

File: students/tonylee/FizzBuzz.py [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+	for number in range(1, 100):
+		if number%3 == 0 and number%5 == 0:
+			print "FizzBuzz"
+
```
Comment (charlesrobison) [inline students/tonylee/FizzBuzz.py]: Go ahead and remove these extra blank rows.

File: students/tonylee/FizzBuzz.py [hunk 3]
```diff
@@ -0,0 +1,13 @@
+def printFizzBuzz():
+	for number in range(1, 100):
+		if number%3 == 0 and number%5 == 0:
+			print "FizzBuzz"
```
Comment (charlesrobison) [inline students/tonylee/FizzBuzz.py]: Looks like this was written for Python 2.7 based on this print statement.  Please use Python 3 for this course.

File: students/tonylee/FizzBuzz.py [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+		
+		elif number%3 == 0:
+			print "Fizz"
+		elif number%5 == 0:
+			print "Buzz"
+		else:
+			print number
+			
+printFizzBuzz()
```
Comment (charlesrobison) [inline students/tonylee/FizzBuzz.py]: Works well, but needs to go all the way to 100 per the assignment.

File: students/tonylee/FizzBuzz.py [hunk 5]
```diff
@@ -0,0 +1,13 @@
+def printFizzBuzz():
+	for number in range(1, 100):
+		if number%3 == 0 and number%5 == 0:
```
Comment (charlesrobison) [inline students/tonylee/FizzBuzz.py]: Typical Python syntax has indentations of 4 spaces.

File: students/tonylee/SecondAssignment.py [hunk 6]
```diff
@@ -0,0 +1,69 @@
+def exchange_first_last(seq):
+    	s = list(seq)
```
Comment (charlesrobison) [inline students/tonylee/SecondAssignment.py]: Convert these indentations to 4 spaces in this file as well.

File: students/tonylee/SecondAssignment.py [hunk 7]
```diff
@@ -0,0 +1,69 @@
...
+	first = seq[0]
+	last = seq[-1]
+	s[0] = last
+	s[-1] = first
+
```
Comment (charlesrobison) [inline students/tonylee/SecondAssignment.py]: Remove extra blank line.

File: students/tonylee/SecondAssignment.py [hunk 8]
```diff
@@ -0,0 +1,69 @@
+def exchange_first_last(seq):
+    	s = list(seq)
+	first = seq[0]
+	last = seq[-1]
+	s[0] = last
+	s[-1] = first
+	
+	return convert_format(seq, s)
```
Comment (charlesrobison) [inline students/tonylee/SecondAssignment.py]: Works well.

File: students/tonylee/SecondAssignment.py [hunk 9]
```diff
@@ -0,0 +1,69 @@
...
+	del s[0:4]
+	del s[len(s)-4:len(s)]
+	
+	return convert_format(seq, s)
+	
+def reverse_element(seq):
+	s = list(seq)
+	r = s[::-1]
+	
+	return convert_format(r)
```
Comment (charlesrobison) [inline students/tonylee/SecondAssignment.py]: This function doesn't seem to be worked correctly.  Getting TypeError: convert_format() missing 1 required positional argument: 'value'

File: students/tonylee/SecondAssignment.py [hunk 10]
```diff
@@ -0,0 +1,69 @@
...
+	length = len(s)
+	third = length/3
+	
+	middle = s[third:length-third]
+	last = s[length-third:length]
+	first = s[0:third]
+	
+	l = middle + last + first
+	
+	return convert_format(seq, l)
```
Comment (charlesrobison) [inline students/tonylee/SecondAssignment.py]: This is returning an exception as well: TypeError: slice indices must be integers or None or have an __index__ method

File: students/tonylee/SecondAssignment.py [hunk 11]
```diff
@@ -0,0 +1,69 @@
+def exchange_first_last(seq):
+    	s = list(seq)
+	first = seq[0]
+	last = seq[-1]
+	s[0] = last
+	s[-1] = first
```
Comment (natasha-aleksandrova) [inline students/tonylee/SecondAssignment.py]: this is using indexing - please use slicing per assignment instructions.

slicing syntax is: [start:end:step]

File: students/tonylee/SecondAssignment.py [hunk 12]
```diff
@@ -0,0 +1,69 @@
...
+
+def convert_format(seq, value):
+	if type(seq) is str:
+		newString = "".join(value)
+		return newString
+		
+	if type(seq) is tuple:	
+		return tuple(value)
+	return value
+
```
Comment (natasha-aleksandrova) [inline students/tonylee/SecondAssignment.py]: @pythonwa please give this exercise another try - you should only use slicing in these and not do any casting to tuples/strings/lists. all of your functions should be pretty much one liners with exception of `mid_last_first`.

please reach out if you need help.

**Pull request merged**